src/modules/displayLikes.js

```diff
@@ -0,0 +1,19 @@
+export async function displayLikes(likes) {
+  // eslint-disable-next-line
+  for (const like of likes) {
```

Comment:
- [ ] Kindly remove the eslint disablement line, and try to solve all eslint errors yourself.